src/utils/navigation-actions.js

```diff
@@ -83,7 +83,7 @@ const navigationActions = [
     navigationCall: async ({ websiteUrl, sameWindow }) => {
       try {
         if (websiteUrl) {
-          const url = websiteUrl.match(/^(https?|mailto):\/\//) ? websiteUrl : `http://${websiteUrl}`;
+          const url = websiteUrl.match(/^(https?:\/\/|mailto:(\/\/)?)/) ? websiteUrl : `http://${websiteUrl}`;
```

Comment:
```suggestion
          const url = websiteUrl.match(/^(https?:\/\/|mailto:)/) ? websiteUrl : `http://${websiteUrl}`;
```

I guess this should be enough as well

Reply:
I'm adding a unit test for mailto as well, so I'll fix that at the same time :)

Reply:
Perfect!